src/ValidationsFactory.js

```diff
@@ -36,6 +36,14 @@ class Validations {
    */
   isVisible() {
     // Disable validations if field is hidden
+    const visibleInDevice =
+      this.element.visibleInDevice === null || this.element.visibleInDevice === undefined
+        ? true
+        : this.element.visibleInDevice;
+    if (!visibleInDevice) {
+      return false;
+    }
+
```

Comment:
Check if this works `const visibleInDevice = !!this.element.visibleInDevice`

Reply:
What if you do `!(!!...)`

Reply:
It won't work because if visibleInDevice has a boolean value it will return the negated (~visibleInDevice), when we want the value of visibleInDevice.